ui/app/src/components/CreateFolderDialog/CreateFolderContainer.tsx

```diff
@@ -63,8 +63,9 @@ export function CreateFolderContainer(props: CreateFolderContainerProps) {
   const itemLookupTable = useItemsByPath();
   const newFolderPath = `${rename ? getParentPath(path) : path}/${name}`;
   const folderExists = rename
-    ? name !== value && itemLookupTable[newFolderPath] !== UNDEFINED
-    : itemLookupTable[newFolderPath] !== UNDEFINED;
+    ? name !== value &&
+      (itemLookupTable[newFolderPath] !== UNDEFINED || itemLookupTable[withIndex(newFolderPath)] !== UNDEFINED)
+    : itemLookupTable[newFolderPath] !== UNDEFINED || itemLookupTable[withIndex(newFolderPath)] !== UNDEFINED;
```

Comment:
There's a útil that checks the same path doing both withIndex and withoutIndex on a given lookup table. Use that one.